Gems/Atom/RPI/Code/Include/Atom/RPI.Public/View.h

```diff
@@ -213,6 +229,9 @@ namespace AZ
 
             // Masked Occlusion Culling interface
             MaskedOcclusionCulling* m_maskedOcclusionCulling = nullptr;
+
+            AZStd::atomic_uint32_t m_andFlags{ 0xFFFFFFFF };
```

Comment:
I think we would benefit from a document laying out how all these interrelated flags/masks are used if we don't already have one...

We have 
PipelineViewTag
DrawListMask/DrawListTags
UsageFlags
AndFlags
OrFlags

plus ShaderOptionFlags on the cullable. I might even be forgetting some.

It's pretty daunting to a new user.

Reply:
Yeah it's a lot, but I don't think there's a lot of interactions, they're used in different places for different things. But I agree some documentation with regards to the flags on the cullable and how they relate to views would be worth doing, probably once the culling system has been through its refactoring.